src/aioswitcher/api/packets.py

```diff
@@ -25,13 +25,59 @@
 
 REQUEST_FORMAT_BREEZE = "{}000001000000000000000000{}00000000000000000000f0fe"
 PAD_72_ZEROS = "0" * 72
-# format value just timestamp (initial session id is "00000000")
+P_SESSION = "00000000"
+PHONE_ID = "0000"
+DEVICE_PASS = "00000000"
+STOP_SHUTTER_PRECOMMAND = "3702"
+SET_POSITION_PRECOMMAND = "3701"
+SET_LIGHT_PRECOMMAND = "370a"
+
+# login packet for Type1 devices
+# format value just timestamp (initial session id is P_SESSION)
 LOGIN_PACKET_TYPE1 = (
-    "fef052000232a10000000000" + REQUEST_FORMAT_TYPE1[2:] + "{}" + PAD_72_ZEROS + "00"
+    "fef052000232a100"
+    + P_SESSION
+    + REQUEST_FORMAT_TYPE1[2:]
+    + "{}"
+    + PAD_72_ZEROS
+    + "00"
+)
+
+# login packet for Type2 devices
+# format values are timestamp, device id
+LOGIN_PACKET_TYPE2 = (
+    "fef030000305a600"
+    + P_SESSION
+    + "ff0301000000"
+    + PHONE_ID
+    + "00000000{}00000000000000000000f0fe{}00"
+)
+
+# login packet for token-based Type2 devices
+# format values are token, timestamp, device id
+LOGIN_TOKEN_PACKET_TYPE2 = (
+    "fef030000305a600"
+    + P_SESSION
+    + "ff0301000000"
+    + "00"
+    + "{}"
+    + "00"
+    + "{}00000000000000000000f0fe{}00"
 )
 
-LOGIN2_PACKET_TYPE2 = (
-    "fef030000305a60000000000ff0301000000000000000000{}00000000000000000000f0fe{}00"
+# second login packet for token-based Type2 devices
+# format values are device id, timestamp, token
+LOGIN2_TOKEN_PACKET_TYPE2 = (
+    "fef053000305a100"
+    + P_SESSION
+    + "f50301000000"
+    + "{}"
+    + "000000"
+    + "{}"
+    + "00000000000000000000f0fe"
+    + "0000"
+    + "{}"
+    + "000000000000000000000000000000000000000000000000000000000000000001"
```

Comment:
Are we sure this won't break?

Reply:
Yes, this is tested.. It is the same end results but with formatted differently to have more sense.